src/components/Button/Button.tsx

```diff
@@ -17,10 +17,10 @@ export type ButtonProps = ButtonHTMLElementProps & {
    * Toggles clickable that fills the full width of its container
    */
   fullWidth?: boolean;
-  status?: ClickableStyleProps<'button'>['status'];
+
```

Comment:
Small thing—did you mean to leave empty lines here where the old props were?

Reply:
Ah good catch 🎣 . I did not. Once we figured out this approach i scrambled to implement it. I suppose prettier won't clean up empty lines on its own in some cases